php-class-splitter.php

```diff
@@ -35,6 +35,20 @@
             }
         } else {
             $code .= $token[1];
+            if ($token[1] === '{') {
+                $braces++;
+            }
+            elseif ($token[1] === '}')
+            {
+                $braces--;
+            }
+            if ($braces == 0) {
+                $buffer = false;
+                $file = $dest . '/' . $name . '.php';
+                $code = '<?php' . PHP_EOL . $code;
+                file_put_contents($file, $code);
+            }
```

Comment:
The block you're adding here seems to duplicate the one on lines 25-34. Maybe try this instead, from line 23 on?

```php
if (is_array($token)) {
  $token = $token[1];
}
if ($token == '{') {
  $braces++;
} elseif ($token == '}') {
  $braces--;
  if ($braces == 0) {
    $buffer = false;
    $file = $dest . '/' . $name . '.php';
    $code = '<?php' . PHP_EOL . $code;
    file_put_contents($file, $code); 
  }
}
```